src/main/java/de/cyface/collector/MainVerticle.java

```diff
@@ -48,23 +48,28 @@ private void prepareEventBus() {
 	}
 
 	private void deployVerticles() {
-		DeploymentOptions options = new DeploymentOptions().setWorker(true);
+		final DeploymentOptions options = new DeploymentOptions().setWorker(true).setConfig(Parameter.MONGO_DATA_DB.jsonValue(vertx, config()));
 		vertx.deployVerticle(SerializationVerticle.class, options);
 	}
 
 	private Router setupRoutes() {
-		String keystorePath = this.getClass().getResource("/keystore.jceks").getFile();
-		JWTAuthOptions config = new JWTAuthOptions()
-				.setKeyStore(new KeyStoreOptions().setPath(keystorePath).setPassword("secret"));
-
-		JWTAuth jwtAuthProvider = JWTAuth.create(vertx, config);
-		AuthHandler jwtAuthHandler = JWTAuthHandler.create(jwtAuthProvider, "/login");
-
-		MongoClient client = SerializationVerticle.createSharedMongoClient(vertx, config());
-		JsonObject authProperties = new JsonObject();
-		MongoAuth authProvider = MongoAuth.create(client, authProperties);
-		List<String> roles = new ArrayList<>();
-		List<String> permissions = new ArrayList<>();
+		final String keystorePath = this.getClass().getResource("/keystore.jceks").getFile();
+		String jwtKeystoreLocation = Parameter.JWT_KEYSTORE.stringValue(vertx, keystorePath);
+		final JWTAuthOptions config = new JWTAuthOptions()
+				.setKeyStore(new KeyStoreOptions().setPath(jwtKeystoreLocation).setPassword("secret"));
```

Comment:
There should be some docu if this pw should be replaced (TODO) or be configurable from outside, ...